docs/ops/server/ml-frameworks/tensorflow.md

```diff
@@ -0,0 +1,50 @@
+<!-- 
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Tensorflow on Kubernetes
+
+## TFJob
+We support to use the tf-operator running the Tensorflow on kubernetes. For more info about tf-operator see [here](https://github.com/kubeflow/tf-operator)
```

Comment:
"We support to use the tf-operator running the Tensorflow on kubernetes" -> "We support Tensorflow job on kubernetes by using the tf-operator as a runtime"